salt/orchestrate/operations/services/fluentd.sls

```diff
@@ -0,0 +1,102 @@
+{% from "orchestrate/aws_env_macro.jinja" import VPC_NAME, VPC_RESOURCE_SUFFIX,
+ ENVIRONMENT, BUSINESS_UNIT, PURPOSE_PREFIX, subnet_ids with context %}
+{% set INSTANCE_COUNT = salt.environ.get('INSTANCE_COUNT', 2) %}
+{% set app_name = 'fluentd' %}
+
+load_{{ app_name }}_cloud_profile:
+  file.managed:
+    - name: /etc/salt/cloud.profiles.d/{{ app_name }}.conf
+    - source: salt://orchestrate/aws/cloud_profiles/{{ app_name }}.conf
+    - template: jinja
+
+generate_{{ app_name }}_cloud_map_file:
+  file.managed:
+    - name: /etc/salt/cloud.maps.d/{{ VPC_RESOURCE_SUFFIX }}_{{ app_name }}_map.yml
+    - source: salt://orchestrate/aws/map_templates/instance_map.yml
+    - template: jinja
+    - makedirs: True
+    - context:
+        environment_name: {{ ENVIRONMENT }}
+        num_instances: {{ INSTANCE_COUNT }}
+        service_name: {{ app_name }}
+        roles:
+          - fluentd-server
+          - log-aggregator
+          - {{ app_name }}
```

Comment:
These roles are already defined in the cloud profile so it is not necessary to duplicate them here.